package/deb-postinst

```diff
@@ -6,15 +6,22 @@ set -e
 case "$1" in
```

Comment:
Same remark about shellcheck.

Reply:
It did not report anything about this file.